.travis.yml

```diff
@@ -1,23 +0,0 @@
-language: node_js
-
-node_js:  
-  - "14"
-
-sudo: required
-
-services:
-  - docker
-
-env:
-  global:
-    # The hostname is set to mongo because of the docker configuration (using networks to access the DB)
-    - DB_HOSTNAME=mongo
```

Comment:
Is this env var no longer needed?

Reply:
Since the tests are passing and I can't find any reference to that in the code, I assume it's no longer needed